Source/Scene/ModelExperimental/PointCloudStylingPipelineStage.js

```diff
@@ -233,11 +255,40 @@ const builtinVariableSubstitutionMap = {
   NORMAL: "attributes.normalMC",
 };
 
-const parameterList = "ProcessedAttributes attributes";
+function getVariableSubstitutionMap(propertyAttributes) {
+  const variableSubstitutionMap = clone(builtinVariableSubstitutionMap);
+
+  if (!defined(propertyAttributes)) {
+    return variableSubstitutionMap;
+  }
+
+  for (let i = 0; i < propertyAttributes.length; i++) {
+    const propertyAttribute = propertyAttributes[i];
+    const properties = propertyAttribute.properties;
+    for (const propertyId in properties) {
+      if (properties.hasOwnProperty(propertyId)) {
+        const variableName = sanitizeGlslIdentifier(propertyId);
```

Comment:
If this code is only ever run for `.pnts` point clouds, then this is redundant, `PntsLoader` already sanitizes the IDs for you